modules/game_mainpanel/mainpanel.lua

```diff
@@ -163,7 +163,7 @@ function optionsController:onInit()
 end
 
 function toggleStore()
-    if g_game.getFeature(GamePurseSlot) then
+    if g_game.getFeature(GamePurseSlot) and g_game.getClientVersion() >= 1332 then
```

Comment:
is this correct? Do you make the store work only for versions 13.32 or above?

Reply:
store only works for +13.32.
as far as I know tfs (13.10) doesn't have a store.

a guy talked to me on discord he had 10.98. (it doesn't have a store either).
in that case I would open the game shop of v8

Reply:
the GamePurseSlot feature is added in version >= 953, that is, its check is: "if version >= 953 AND version >= 1332 ... "